src/accounts/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import Wallet from "./wallet";
+import axios from 'axios';
+import AddWalletModal from "../modals/addWallet";
+import Loader from "../components/shared/Loader";
+import Swal from "sweetalert2";
+
+import "./style.scss"
+
+export default function Accounts() {
+    const [walletData, setWalletData] = useState([]);
+    const [modalIsOpen, setModalIsOpen] = useState(false)
+    const [getNewData, setGetNewData] = useState(false)
+    const [isLoaded, setIsLoaded] = useState(true)
+
+    function ErrorAlert() {
```

Comment:
make it camel case with starting lower case. And make not to be function but arrow function